site/cds_rdm/files.py

```diff
@@ -45,12 +92,18 @@ def send_file(
             and current_app.config["FILES_REST_XSENDFILE_ENABLED"]
         )
 
+        if not offload_enabled:
+            # don't offload if not enabled
+            return super().send_file(filename, **kwargs)
```

Comment:
Nothing to do with your changes, but I just noticed that here we don't pass the explicit kwargs (mimetype, ...). Can you check the upper send_file if it expects them?

Reply:
done, added